### PR TITLE
[DOCS] Updating datasource docs to remove sample data output and general cleanup

### PR DESCRIPTION
**Pull request by spbail**
Changes proposed in this pull request:
- Removed example output to match code changes
- Some general cleanup to match our style guide for spelling and conciseness of "how to" guides
- Added a v3 API guide for BigQuery!

### CODE REVIEW

File: docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_mssql_datasource.rst [hunk 1]
```diff
@@ -193,21 +190,9 @@ Steps
...
-                    [(58098,)]
 
-                            Showing 5 rows
-                       movieId                               title                                         genres
-                    0        1                    Toy Story (1995)  Adventure|Animation|Children|Comedy|Fantasy\r
-                    1        2                      Jumanji (1995)                   Adventure|Children|Fantasy\r
-                    2        3             Grumpier Old Men (1995)                               Comedy|Romance\r
-                    3        4            Waiting to Exhale (1995)                         Comedy|Drama|Romance\r
-                    4        5  Father of the Bride Part II (1995)                                       Comedy\r
+            This means all has gone well and you can proceed with configuring your new Datasource.             If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```
Comment (anthonyburdi) [inline docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_mssql_datasource.rst]:
```suggestion
            This means all has gone well and you can proceed with configuring your new Datasource. If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```

File: docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_configuredassetdataconnector.rst [hunk 2]
```diff
@@ -9,7 +9,7 @@ This guide demonstrates how to configure a ConfiguredAssetDataConnector, and pro
 
     - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
     - :ref:`Understand the basics of Datasources in 0.13 or later <reference__core_concepts__datasources>`
-    - Learned how to configure a :ref:`DataContext using test_yaml_config <how_to_guides_how_to_configure_datacontext_components_using_test_yaml_config>`
+    - Learned how to configure a :ref:`Data Context using test_yaml_config <how_to_guides_how_to_configure_datacontext_components_using_test_yaml_config>`
```
Comment (anthonyburdi) [inline docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_configuredassetdataconnector.rst:12]: Thanks!!

File: docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_mysql_datasource.rst [hunk 3]
```diff
@@ -190,21 +186,7 @@ Steps
...
-                        Showing 5 rows
-                   movieId                               title                                         genres
-                0        1                    Toy Story (1995)  Adventure|Animation|Children|Comedy|Fantasy\r
-                1        2                      Jumanji (1995)                   Adventure|Children|Fantasy\r
-                2        3             Grumpier Old Men (1995)                               Comedy|Romance\r
-                3        4            Waiting to Exhale (1995)                         Comedy|Drama|Romance\r
-                4        5  Father of the Bride Part II (1995)                                       Comedy\r
-
-             This means all has went well and you can proceed with exploring data in your new MySql datasource.
+            This means all has gone well and you can proceed with configuring your new Datasource.             If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```
Comment (anthonyburdi) [inline docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_mysql_datasource.rst]:
```suggestion
            This means all has gone well and you can proceed with configuring your new Datasource. If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```

File: docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_pandas_filesystem_datasource.rst [hunk 4]
```diff
@@ -205,16 +206,10 @@ Steps
...
                     Unmatched data_references (0 of 0): []
 
-                    Choosing an example data reference...
-                        Reference chosen: abe_20201119_200.csv
-
-                        Fetching batch data...
-
-                        Showing 5 rows
-                        ...
+            This means all has gone well and you can proceed with configuring your new Datasource.             If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```
Comment (anthonyburdi) [inline docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_pandas_filesystem_datasource.rst]:
```suggestion
            This means all has gone well and you can proceed with configuring your new Datasource. If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```

File: docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_pandas_s3_datasource.rst [hunk 5]
```diff
@@ -207,27 +206,11 @@ Steps
 
                     Unmatched data_references (0 of 0): []
 
-                    Choosing an example data reference...
-                        Reference chosen: abe_20201119_200.csv
 
-                        Fetching batch data...
+            This means all has gone well and you can proceed with configuring your new Datasource.             If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```
Comment (anthonyburdi) [inline docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_pandas_s3_datasource.rst]:
```suggestion
            This means all has gone well and you can proceed with configuring your new Datasource. If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```

File: docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_redshift_datasource.rst [hunk 6]
```diff
@@ -229,29 +229,8 @@ Steps
...
-
-            If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.  Whenever possible, ``test_yaml_config`` provides helpful warnings and error messages, like the example below. It can't solve every problem, but it can solve many.
-
-            .. code-block:: bash
-
-                ...
 
-                psycopg2.OperationalError: FATAL:  password authentication failed for user "my_username"
-                FATAL:  password authentication failed for user "my_username"
+            This means all has gone well and you can proceed with configuring your new Datasource.             If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```
Comment (anthonyburdi) [inline docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_redshift_datasource.rst]:
```suggestion
            This means all has gone well and you can proceed with configuring your new Datasource. If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```

File: docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_snowflake_datasource.rst [hunk 7]
```diff
@@ -274,21 +272,8 @@ Steps
...
-                Showing 5 rows
-               movieid                               title                                       genres
-                0        1                    Toy Story (1995)  Adventure|Animation|Children|Comedy|Fantasy
-                1        2                      Jumanji (1995)                   Adventure|Children|Fantasy
-                2        3             Grumpier Old Men (1995)                               Comedy|Romance
-                3        4            Waiting to Exhale (1995)                         Comedy|Drama|Romance
-                4        5  Father of the Bride Part II (1995)                                       Comedy
 
-            This means all has went well and you can proceed with exploring data with your new Snowflake datasource.
+            This means all has gone well and you can proceed with configuring your new Datasource.             If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```
Comment (anthonyburdi) [inline docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_snowflake_datasource.rst]:
```suggestion
            This means all has gone well and you can proceed with configuring your new Datasource. If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```

File: docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_spark_filesystem_datasource.rst [hunk 8]
```diff
@@ -196,19 +193,8 @@ Steps
...
-
-                        Showing 5 rows
-                0  None                                           Name  PClass  Age     Sex  Survived  SexCode
-                1     1                   Allen, Miss Elisabeth Walton     1st   29  female         1        1
-                2     2                    Allison, Miss Helen Loraine     1st    2  female         0        1
-                3     3            Allison, Mr Hudson Joshua Creighton     1st   30    male         0        0
-                4     4  Allison, Mrs Hudson JC (Bessie Waldo Daniels)     1st   25  female         0        1
 
-            This means all has gone well and you can proceed with exploring data with your new filesystem-backed Pandas data source.
+            This means all has gone well and you can proceed with configuring your new Datasource.             If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```
Comment (anthonyburdi) [inline docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_spark_filesystem_datasource.rst]:
```suggestion
            This means all has gone well and you can proceed with configuring your new Datasource. If something about your configuration wasn't set up correctly, ``test_yaml_config`` will raise an error.
```

**Review by anthonyburdi — APPROVED** [review]
LGTM! Approved with the suggested spacing changes.

**Comment by spbail** [comment]
Thanks for fixing my spacing #smallscreenproblems

**Pull request merged**